### PR TITLE
feat:added sidebar

### PR DESCRIPTION
**Pull request by Developers-codz**
Added sidebar for smaller device 
Added home and shop navigation link in header.

Do review and give your valuable feedbacks. thank you :)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -166,11 +184,40 @@ footer{
...
+        top:0;
+        left:-50%;
+        background-color: var(--bg-off-white);
+        z-index: 3;
+        width:50%;
+        min-height:100vh;
+        font-size:2rem;
+        transition: all 500ms linear;
+    }
+    .navbar-items>li{
```
Comment (VLeads) [inline style.css:213]: Looking good. Also add cursor pointer to list item and cancel button.

**Review by VLeads — CHANGES_REQUESTED** [review]

**Review by vrinda-mahajan — APPROVED** [review]
The code looks good. But your 2nd and 3rd image inside the carousel is overflowing the grid.

File: style.css (hunk 1 above)
Comment (Developers-codz) [reply style.css:213]: Done! thanks for reviewing :)

**Comment by Developers-codz** [comment]
> The code looks good. But your 2nd and 3rd image inside the carousel is overflowing the grid.

is it overflowing? I think this is because of image size I will fix it. Thanks for reviewing :)

**Pull request merged**